packages/account/src/Components/self-exclusion/self-exclusion-inputs.jsx

```diff
@@ -5,7 +5,7 @@ import { getBrandWebsiteName, epochToMoment, toMoment, PlatformContext, isMobile
 import { Localize, localize } from '@deriv/translations';
 import { Field, useFormikContext } from 'formik';
 import SelfExclusionContext from './self-exclusion-context';
-import SelfExclusionFooter from './self-exclusion-footer.jsx';
+import SelfExclusionFooter from './self-exclusion-footer';
```

Comment:
I believe we should also add types for all context variables in self-exclusion-footer.tsx

Reply:
yes.there are some ts errors in the file.

Reply:
Done Updating the codes 🙏